app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -98,24 +98,48 @@ public void onClickFrom(View v){
     public void onClickTransit(View v) {
         mViewModel.setTransportType(ClassConstants.TRANSIT);
         getAllRoutes();
+
+        transitButton.setSelected(true);
+        shuttleButton.setSelected(false);
+        walkButton.setSelected(false);
+        carButton.setSelected(false);
     }
 
     public void onClickCar(View v) {
         mViewModel.setTransportType(ClassConstants.DRIVING);
         getAllRoutes();
+
+        transitButton.setSelected(false);
+        shuttleButton.setSelected(false);
+        walkButton.setSelected(false);
+        carButton.setSelected(true);
     }
 
+    // TODO: #180
     public void onClickDisability(View v) {
-
+        if(disabilityButton.isSelected())
+            disabilityButton.setSelected(false);
+        else
+            disabilityButton.setSelected(true);
     }
 
     public void onClickShuttle(View v) {
-        getShuttle(v);
+        //getShuttle(v);
```

Comment:
remove it if it isnot needed anymore

Reply:
It causes crashes but it is needed. @CarenRizk will have to look into it.